app/controllers/forms/delete_confirmation_controller.rb

```diff
@@ -80,8 +79,7 @@ def delete_page(form, page)
 
       authorize form, :can_view_form?
       if page.destroy
-        flash[:message] = "Successfully deleted #{page.question_text}"
-        redirect_to success_url, status: :see_other
+        redirect_to success_url, status: :see_other, success: "Successfully deleted ‘#{page.question_text}‘"
```

Comment:
I think this is two left curly quotes, it should be:

```suggestion
        redirect_to success_url, status: :see_other, success: "Successfully deleted ‘#{page.question_text}’"
```